microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -381,21 +381,19 @@ The family group organizer may stop the data collection from the family safety p
 
 
 <!-- ====================================================================== -->
-## Find on Page 
+## Find on page 
 
-Find on Page allows you to search a web page for important keywords. Open **Settings and more (...)** > **Find on Page**. A search box will appear. Type any keyword or phrase into this search box and the page will highlight all locations on the page where your exact search term appears. You can then move through these terms with the **Previous result** and **Next result** buttons. 
+Find on page allows you to search a web page for important keywords.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**. A search box opens. Type a word or phrase into this search box, and the page highlights all locations on the page where your search string appears. You can then move through these terms by clicking the **Previous result** and **Next result** buttons.
```

Comment:
```suggestion
Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.
```
* "webpage": 1 word per corp style guide.  https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/web-world-wide-web-www
* "Use the ..." - per corp style guide, instead of "allows". https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows (+ other pages).
* "click" - per internal Edge Dev Style Guide & this repo's conventions.